android/assets/jsons/Tutorials.json

```diff
@@ -168,5 +168,53 @@
-  ]
+  ],
+    "Faith": [
+        "After building a shrine, you're civilization will start generating faith.",
+        "When enough faith has been generated, you will be able to found a pantheon.",
+        "A pantheon will provide a small bonus for your civilization that will apply to all your cities.",
+        "Each civilization can only choose a single pantheon belief, and each pantheon can only be chosen once.",
+        "Generating more faith will allow you to found a religion."
+    ],
+    "Religion": [
+        "When you have gathered enough religion, a great prophet will be born in one of your cities.",
+        "This great prophet can be used for multiple things: Constructing a holy city, founding a religion and spreading your religion.",
+        "When founding your religion, you may choose another two beliefs. The founder belief will only apply to you, while the follower belief will apply to all cities following your religion.",
+        "Additionally, the city where you used your great prophet will become the holy city of that religion.",
+        "If after founding your religion another great prophet is born in one of your cities, you can use him to enhance your religion.",
+        "This will allow you to choose another follower belief, as well as an enhancer belief, that only applies to you."
+    ],
+    "Beliefs": [
+        "There are four types of beliefs: Pantheon, Founder, Follower and Enhancer beliefs.",
+        "Pantheon and Follower beliefs apply to each city following your religion, while Founder and Enhancer beliefs only apply to the founder of a religion."
+    ],
+    "Spreading_Religion": [
+        "When founding a city, it won't follow a religion immediately.",
+        "The religion a city follows depends on the total pressure each religion has within the city.",
+        "Followers are allocated in the same proportions as these pressures, and these followers can be viewed in the city screen.",
+        "Based on this, you can get a feel for which religions have a lot of pressure build up in the city, and which have almost none.",
+        "The city follows a religion if a majority of its population follows the city, and will only than be effected by Follower and Pantheon beliefs of that religion.",
+        "",
+        "Spreading religion happens naturally, but can be sped up using missionaries or great prophets.",
+        "Missionaries can be bought in cities following a major religion, and take the religion of that city.",
+        "So do take care where you are buying them! If another civilization has converted one of your cities to their religion, missionaries bought their will follow their religion.",
+        "Great prophets always have your religion, regardless if they are bought or spawned naturally, but captured great prophets do retain their original religion.",
```

Comment:
Somehow that regardless doesn't quite fit together ... "no matter whether they are bribed or hellspawn" perhaps? _Can_ one still capture Great Prophets?